packages/aws-cdk/lib/api/evaluate-cloudformation-template.ts

```diff
@@ -1,6 +1,55 @@
 import * as cxapi from '@aws-cdk/cx-api';
 import * as AWS from 'aws-sdk';
-import { ListStackResources } from './common';
+import { ISDK } from './aws-auth';
+
+export async function establishResourcePhysicalName(
```

Comment:
Can we make this a method of `EvaluateCloudFormationTemplate` instead of a stand-alone function?

Reply:
Done.